x11-dl/src/link.rs

```diff
@@ -55,12 +55,11 @@ macro_rules! x11_link {
         unsafe {
           let libdir = $crate::link::config::libdir::$pkg_name;
           let lib = try!($crate::link::DynamicLibrary::open_multi(libdir, &[$($lib_name),*]));
-          let mut this: ::std::mem::ManuallyDrop<$struct_name>
-              = ::std::mem::uninitialized();
+          let mut this = ::std::mem::MaybeUninit::<$struct_name>::uninit();
           let this_ptr = &mut this as *mut _ as *mut $struct_name;
```

Comment:
Now that we're using `MaybeUninit`, it might be better to use `this.as_mut_ptr()` here